README.md

```diff
@@ -176,7 +160,48 @@ docker-compose -f docker-compose.prod.yml up --build
 After running for the first time, you can omit build to skip the build stage.
 
 ### Running the development environment
-The development environment runs the application and tests. The tests will run every time a file change is detected. The Flask server will also restart when changes are detected.
+The development environment runs the application and tests. The tests will run every time a file change is detected. The Flask server will also restart when changes are detected. It will also run the dev database which runs in a separate container.
+
+It is recommended to create another .env file for running the development environment. This allows a dev database to be used. Create a new .env file called `.env.docker.dev`:
+
+```bash
+FLASK_APP=todo_app/app
+FLASK_RUN_HOST=0.0.0.0
+FLASK_ENV=docker
+FLASK_RUN_PORT=5000
+
+SECRET_KEY=secret-key
+
+KEY_VAULT_NAME=todo-app
+
+MONGO_CONNECTION_STRING=mongodb://[dev-db-username]:[dev-db-password]@[mongo-container-name]:27017/
+MONGO_DATABASE_NAME="todo-db"
+MONGO_INITDB_ROOT_USERNAME=[dev-db-username]
+MONGO_INITDB_ROOT_PASSWORD=[dev-db-password]
```

Comment:
Would we be better off just specifying these within the docker-compose file itself? Or at least, just including this `.env.docker.dev` file as part of the repo, since it can just be unimportant local test credentials that get never re-used on a deployed system?